alignak/objects/config.py

```diff
@@ -2488,68 +2397,80 @@ def create_packs(self, nb_packs):  # pylint: disable=R0915,R0914,R0912,W0613
 
+        logger.info("Splitting configuration into parts")
+
         # We create dummy configurations for schedulers:
-        # they are clone of the master
-        # conf but without hosts and services (because they are dispatched between
-        # theses configurations)
+        # they are clone of the master configuration but without hosts and services
+        # (because they are dispatched between these configurations)
         self.confs = {}
-        for i in xrange(0, nb_parts):
-            # print "Create Conf:", i, '/', nb_parts -1
-            cur_conf = self.confs[i] = Config()
+        for index in xrange(0, nb_parts):
+            current_conf = self.confs[index] = Config({'name': "part-%d" % index})
+            logger.debug("Creating a new configuration: %s", current_conf)
 
-            # Now we copy all properties of conf into the new ones
+            # Now we copy all properties of a configuration into the newly created
             for prop, entry in Config.properties.items():
+                if prop in ['uuid', 'name']:
+                    # Do not change uuid nor configuration name
+                    continue
                 if entry.managed and not isinstance(entry, UnusedProp):
-                    val = getattr(self, prop)
-                    setattr(cur_conf, prop, val)
-                    # print "Copy", prop, val
+                    setattr(current_conf, prop, getattr(self, prop))
 
             # we need a deepcopy because each conf
             # will have new hostgroups
-            cur_conf.uuid = uuid.uuid4().hex
-            cur_conf.commands = self.commands
-            cur_conf.timeperiods = self.timeperiods
-            # Create hostgroups with just the name and same id, but no members
-            new_hostgroups = []
-            for hostgroup in self.hostgroups:
-                new_hostgroups.append(hostgroup.copy_shell())
-            cur_conf.hostgroups = Hostgroups(new_hostgroups)
-            cur_conf.notificationways = self.notificationways
-            cur_conf.checkmodulations = self.checkmodulations
-            cur_conf.macromodulations = self.macromodulations
-            cur_conf.businessimpactmodulations = self.businessimpactmodulations
-            cur_conf.resultmodulations = self.resultmodulations
-            cur_conf.contactgroups = self.contactgroups
-            cur_conf.contacts = self.contacts
-            cur_conf.triggers = self.triggers
-            cur_conf.escalations = self.escalations
-            # Create hostgroups with just the name and same id, but no members
-            new_servicegroups = []
-            for servicegroup in self.servicegroups:
-                new_servicegroups.append(servicegroup.copy_shell())
-            cur_conf.servicegroups = Servicegroups(new_servicegroups)
-            # Create ours classes
-            cur_conf.hosts = Hosts([])
-            cur_conf.services = Services([])
+
+            # current_conf.uuid = uuid.uuid4().hex
```

Comment:
it seems that the config is not differentiated by its uuid but with its magic_hash or flavor ... to be confirmed by @ddurieux